python/felis/datamodel.py

```diff
@@ -513,6 +513,11 @@ class CheckConstraint(Constraint):
     expression: str
     """Expression for the check constraint."""
 
+    @field_serializer("type")
+    def serialize_type(self, value: str) -> str:
+        """Ensure '@type' is included in serialized output."""
+        return value
```

Comment:
I don't understand why this method is necessary.  Wouldn't pydantic save it even if it's not present?

Reply:
I don't fully understand it either, but Pydantic was ignoring the `type` field without this.

Reply:
(As discussed, this is because I'm excluding default values when serializing.)